### ISSUE TITLE
Dot marker activity for penmanship & fine motor control

### ISSUE THREAD

**Issue by iteles** [bmhco/shared#44]
When A 'draws', this is currently just a random scribble, irrespective of if he is trying to draw something, trying to colour something in or generally doodling. 

I'm keen to have a LOT of different activities for him to encourage him to have a little more control over his use of a drawing implement, allowing him to practice a more deliberate line which he will eventually need for writing. 

One of these activities is using dot markers. 

We already have the dot markers. Now we just want a targeted activity for its use.  

This is the kind of thing I am thinking of:

https://busybox-montessori.jumpseller.com/busy-book-digital-i-trabalha-com-as-tuas-maos

![image](https://user-images.githubusercontent.com/4185328/233504639-d2253ccf-1091-407e-82b4-bc4e3f343d20.jpeg)

@Stephanymtr Please find some images of animals or objects of interest to Alex that have either been pre-prepared with the white circles for use with dot markers or that we can add white circles to. 
Ideally they would be as un-cartoony as possible but we will still need them to be somewhat cartoony - we want them to be all one colour (two at most for a later stage) so that it looks 'right' to A when it's filled in. 
We'll have to trial and error the size of the circles too if using our own images.

**Comment by Stephanymtr** [comment]
Hi, I couldn't find pre-prepared relevant images with the white circles. 
I used Fotor to add the white circles. I spent decent amount of time looking for a free red only (or red and black) firetruck and failed. Since there will be a trial and error, I figured to play around with the images/circles sizes. Let me know if the images below have too many colors/shades. It's tricky to find un-cartoony with 1 or 2 colors.

Basketball image from : https://www.freepik.com/free-vector/basketball-ball-isolated_10603057.htm#query=basketball&position=0&from_view=keyword&track=sph

Music Instrument/ Violin image from: https://www.pngwing.com/en/free-png-bougq

with white circles:
![basketball with white cirlcles](https://user-images.githubusercontent.com/91985721/233594883-bb706cc9-6bd5-4f9f-b8ea-07fcf5977fbf.png)
![violin white circles](https://user-images.githubusercontent.com/91985721/233594949-5f13307d-01e2-4904-b046-d287f970ed54.png)



Screenshoots as a reference for myself for later:
<img width="1440" alt="Screenshot 2023-04-21 at 09 58 48" src="https://user-images.githubusercontent.com/91985721/233594708-82dbe989-62d2-49b6-9190-f544acf2e670.png">
<img width="1440" alt="Screenshot 2023-04-21 at 10 03 00" src="https://user-images.githubusercontent.com/91985721/233594713-d108d628-4cc9-4222-96d0-e40aaa817035.png">

**Comment by iteles** [comment]
Yes! The only annoying thing about making them ourselves is that we have to print and check the size of the dots and keep correcting them until they are the right size.

But I googled for 'dot marker activity' and found these that might have some good ones like apple, bear, carrots and duck?
http://homeschoolcreations.com/files/ABC_Do_a_Dots_A-M.pdf
http://homeschoolcreations.com/files/ABC_Do_a_Dots_N_-_Z.pdf

Not ALL of them are acceptable so we'd have to pick and choose a few.

I got there from here: [Montessori Do-A-Dot Marker Activities & Free Printables!](https://simplymontessori.blogspot.com/2010/11/do-dot-marker-activities-free.html?m=1)
Which I got to from: [Tons of free do a dot printables](https://homeschoolgiveaways.com/2020/11/tons-of-free-dab-a-dot-printables/)

**Comment by Stephanymtr** [comment]
Great! I screenshotted a couple of the ones I think could be used. Let me know if (or which ones) I can print. Would you like each of the drawings/images to take up an entire A4 page? 

Using that same Google query, I was able to find these too https://www.giftofcuriosity.com/downloads/Insect_Do-A-Dot_Printables.pdf

<img width="492" alt="Screenshot 2023-04-28 at 10 33 46" src="https://user-images.githubusercontent.com/91985721/235114252-72bde015-73ae-4f74-be46-5eb1d6aa1f06.png">
<img width="485" alt="Screenshot 2023-04-28 at 10 33 22" src="https://user-images.githubusercontent.com/91985721/235114273-0b482762-7629-4be3-8d09-f7f1245e3229.png">
<img width="514" alt="Screenshot 2023-04-28 at 10 41 18" src="https://user-images.githubusercontent.com/91985721/235114095-c104b3ef-2194-4e35-b2a6-df96023aaa63.png">
<img width="527" alt="Screenshot 2023-04-28 at 10 40 45" src="https://user-images.githubusercontent.com/91985721/235114114-c1ff739f-67ba-47cc-baf6-a226abfcd23a.png">
<img width="457" alt="Screenshot 2023-04-28 at 10 39 52" src="https://user-images.githubusercontent.com/91985721/235114137-af95fae2-7e01-4073-9940-b706f07cb4d8.png">
<img width="544" alt="Screenshot 2023-04-28 at 10 39 22" src="https://user-images.githubusercontent.com/91985721/235114169-4b08a0ab-c270-463f-ac62-9f6a06f0e951.png">
<img width="508" alt="Screenshot 2023-04-28 at 10 36 50" src="https://user-images.githubusercontent.com/91985721/235114204-f0f5245e-f82d-4b37-8e87-d955fc84462f.png">
<img width="465" alt="Screenshot 2023-04-28 at 10 35 58" src="https://user-images.githubusercontent.com/91985721/235114236-5378f6cd-3756-41c5-9191-da210497d648.png">

**Comment by iteles** [comment]
Yea, I think the point of the way they were on the PDF is that they were that the dots were the 'right size' for the dot markers so we should use that sizing as our guide. Seeing as we have that sizing guide, we could totally just do our own:
+ Keep the ratio on the A4 page as per the pdf
+ Get our own images on the page at the same size
+ Draw a dot over the existing dot on one of the drawings in the PDF
+ Copy paste that dot to our images as many times as we want

I think we could even start our own ones off pretty simple, with only 4-5 dots tops so that there's more room for error when he misses a dot.

Images like:
https://i0.wp.com/www.gardeningchannel.com/wp-content/uploads/2020/10/Gardman-8327-Hunter-Green-Galvanized-Steel-Watering-Can-with-Copper-Accents-1-Gallon.jpg?resize=1536%2C1266&ssl=1
![image](https://user-images.githubusercontent.com/4185328/235167202-52af36f9-6a3e-42a5-93b3-f85eea433769.png)

https://spinnaker-one.fr/modules/ps_imageslider/images/432a3282357859f2af685b127277adb0648e0abd_SPINAKKER_RESIZE_5-2.jpg
<img width="734" alt="image" src="https://user-images.githubusercontent.com/4185328/235168680-f44bfaed-8a09-44fa-911a-72920da22f17.png">

***We could actually do a whole activity on boats and water vehicles - spinnakers on yachts are perfect for this activity!***
https://i.pinimg.com/originals/bb/20/fb/bb20fbb13eb50d35f3e6c8c97e08a307.jpg
![image](https://user-images.githubusercontent.com/4185328/235169002-eb304b06-5ca6-420f-9fe3-2abb68894c96.png)

**Comment by Stephanymtr** [comment]
Sharing my progres for feedback. Please, ignore first and last page - They are for reference.

https://docs.google.com/document/d/1BCM_h29VMSKzLDZgVjndb4IHvNTT3xepYJDUGFQ_cKs/edit

**Comment by Stephanymtr** [comment]
Hi, I added some more photos to the file. We have 3 sections: gardening, sports and water vehicles.
